lib/enhanceReport.js

```diff
@@ -147,8 +147,15 @@ function createVideoContext(video, baseFolder) {
 
 function convertImageToBase64(screenshotsDir, imagePath) {
   const imgPath = path.join(screenshotsDir, imagePath);
```

Comment:
it looks like `imagePath`  in your case is an absolute path, then it adds another absolute path, so you can add a check and if the variable is already absolute, use it as it is.

what do you think? it might solve the issue

If you add this part, I think you can revert the other parts, because `imagePath` won't need to be inside `screenshotsDir`, and the current NodeJS exception is pretty good.

```suggestion
  const imgPath = path.isAbsolute(imagePath) ? imagePath : path.join(screenshotsDir, imagePath);
```

Reply:
thanks for the code review @LironEr i pushed your comment ! Thanks!

Reply:
@LironEr i notice now my latest commit i removed the main part of the function convertImageToBase64() (whoops) i am adding it back but im testing it now and it is still failing for me with 
```
Error: ENOENT: no such file or directory, open '\screenshot-compare.e2e.cy.ts\screenshot-compare.e2e.cy.ts -- users page - assert img (failed).png'
    at Object.openSync (node:fs:596:3)
    at Object.readFileSync (node:fs:464:35)
    at convertImageToBase64 (C:\dev\frontend-e2e-cypress\node_modules\cypress-mochawesome-reporter\lib\enhanceReport.js:164:52)
    at C:\dev\frontend-e2e-cypress\node_modules\cypress-mochawesome-reporter\lib\enhanceReport.js:147:13
    at Array.map (<anonymous>)
```


```
======imgPath         C:\dev\frontend-e2e-cypress\cypress\snapshots\screenshot-compare\screenshot-compare.e2e.cy.ts\__diff_output__\user list.diff.png
=====path.isAbsolute(imagePath)true


======imgPath     \screenshot-compare.e2e.cy.ts\screenshot-compare.e2e.cy.ts -- users page - assert img (failed).png
=====path.isAbsolute(imagePath)true
```

Reply:
it looks like `context.screenshots.value[]` has this 
```
\screenshot-compare.e2e.cy.ts\\\\screenshot-compare.e2e.cy.ts -- users page - assert img (failed).png
```
which is a valid `path.isAbsolute()` according to https://www.w3schools.com/nodejs/met_path_isabsolute.asp..... context seems to not pass in the cypress screenshots dir in the path. would this be a bad idea? (code below) I just dont know what to do if nothing does exist

```
function convertImageToBase64(screenshotsDir, imagePath) {
  const imgPath = path.isAbsolute(imagePath) ? imagePath : path.join(screenshotsDir, imagePath);
  if (fse.pathExistsSync(imgPath)) {
    const contents = 'data:image/png;base64, ' + fse.readFileSync(imgPath, { encoding: 'base64' });
    return contents;
  }
}
```

Reply:
@LironEr bump baybay bump